slometricsemitter/log_processor.go

```diff
@@ -198,20 +201,24 @@ func (sloMetricsProc *sloMetricsProcessor) getAttributeFromLogRecord(logRecord p
 	return attrVal.Str(), nil
 }
 
-func (sloMetricsProc *sloMetricsProcessor) determineEndpointType(logRecord plog.LogRecord) (string, error) {
+// Defaults to http if content_type attribute is missing or unknown.
```

Comment:
Is there a reason you chose to default to `http` here instead of some `unknown` value? is this a common situation?

Reply:
i actually just copied what we do in [latency reporter](https://github.com/Affirm/all-the-things/blob/master/jvm-o11y-kt/latency-reporting-flink/src/main/kotlin/aff/o11yKt/latencyReporting/common/EnvoyAccessLogDeserializer.kt#L90-L94)

Reply:
i'll look into this and when it actually happens